mesheryctl/Makefile

```diff
@@ -2,40 +2,33 @@ make:
 	go build -o mesheryctl cmd/mesheryctl/main.go
 forwin:
 	go build -o mesheryctl.exe cmd/mesheryctl/main.go
```

Comment:
@Aisuko @Nikhil-Ladha @nunix @vijcheru Does the `make forwin` work on your Windows machines?

Reply:
just built it from Powershell after I installed gnumake and golang.
Everything went OK as far as I can tell 👍

Reply:
`forwin` is work for me.